### DISCUSSION TITLE
using Latex axes labels

### DISCUSSION THREAD

**Discussion by paulnakroshis** [cjdoris/Bokeh.jl#23] [Q&A]
I have tried to use LaTeX in Bokeh.jl to label my axes, but have not been successful. Can anyone provide a working example?

**Comment by cjdoris** [comment]
On my phone so can't check, but it should be as simple as passing `raw"$$\some\tex$$"` as the label. If that doesn't work try `TeX(text=raw"\some\tex")`.
